cpp/src/Ice/CommunicatorI.cpp

```diff
@@ -377,6 +375,12 @@ Ice::CommunicatorI::getServerDispatchQueue() const
 
 #endif
 
+void
+Ice::CommunicatorI::postToClientThreadPool(function<void()> call)
```

Comment:
I'm not sure what the correct term I supposed to be for this sort of operation, but this seems to be the only place we use post. Maybe `enqueueOnCLientThreadPool`